docs/tutorials/05-model-monitoring.ipynb

```diff
@@ -282,9 +290,17 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Deploying Evidently based app\n",
+    "To retrieve a list of the model monitoring functions, run [list_model_monitoring_functions](https://docs.mlrun.org/en/stable/api/mlrun.projects/index.html#mlrun.projects.MlrunProject.list_model_monitoring_functions).\n",
+    "For example: `functions = project.list_model_monitoring_functions()`,"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Deploying Evidently-based app\n",
     "You can use the MLRun built-in class, `EvidentlyModelMonitoringApplicationBase`, to integrate [Evidently](https://github.com/evidentlyai/evidently) as an MLRun function and create MLRun artifacts.  \n",
-    "As prerequsite, you need to have an Evidently workspace and project. If you don't have one, create it now:"
+    "As a prerequsite, you need to have an Evidently workspace and project. If you don't have one, create it now:"
```

Comment:
I saw that after this cell we call again
`from mlrun.model_monitoring.applications.evidently import SUPPORTED_EVIDENTLY_VERSION`
which is not necessary because we already imported it at the beginning to install the relevant evidently package